### ISSUE TITLE
Getting empty assets array on video pick from gallery ios(15.2)[🐛] 

### ISSUE THREAD

**Issue by vishal-bhil** [react-native-image-picker/react-native-image-picker#1945]
const selectVideoFromGallery = () => {
    const options = {
      title: 'Video Picker',
      mediaType: 'video',
      storageOptions: {
        skipBackup: true,
        path: 'images',
      },
    };
    launchImageLibrary(options, (response) => {
      console.log('launchImageLibrary->', response);
    });
  };

<img width="355" alt="Screenshot 2022-02-22 at 6 29 25 PM" src="https://user-images.githubusercontent.com/34021048/155137616-701b5987-b5c4-4156-b59e-898b60ff01ca.png">

**Comment by Johan-dutoit** [comment]
Is this reproducible with the example app?

**Comment by vishal-bhil** [comment]
> Is this reproducible with the example app?

Yes

**Comment by SherifSamir5** [comment]
I got the same Issue, is it only for emulators ?

**Comment by SherifSamir5** [comment]
I even tried with IOS 15.0 and got the same issue
 LOG  -> {"assets": []}

**Comment by BhargaviOS** [comment]
correct. getting blank access array while pick up videos from launchImageLibrary.

**Comment by trevyd23** [comment]
getting the same issue

**Comment by SherifSamir5** [comment]
After 2 weeks I discovered that this behavior is only on simulators, as I have tried the same thing on my iPhone and it is working properly !

**Comment by dehimer** [comment]
Same for me in simulator. iPhone 12 (iOS 15.4)

**Comment by jittuu** [comment]
For me, it is working in simulator (iOS 15.4)  ✅
but **not** working on device (iOS 15.4.1) ❌

**Comment by jonasgroendahl** [comment]
getting same issue on simulator IOS 15.4. Works on device

**Comment by arvind-bizburd** [comment]
Is there any update on this issue?
I am also getting same issue on simulator as well as on device with iOS 15.5

**Comment by JungHsuan** [comment]
getting same issue on simulator iOS 15.5, works fine on iOS 15.5 device.

**Comment by luis-antonio-dev** [comment]
iOS 15.4, iPhone SE 3rd generation, same issue
